test/expectations/inline-source-map.js

```diff
@@ -0,0 +1,11 @@
+"format register";
+
+(function() {
+function define(){};  define.amd = {};
+System.register("tree/amd-2", [], false, function() {
+  return {amd: '2'};
+});
+
+
+})();
+//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInRyZWUvYW1kLTIuanMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7Ozs7T0FPTyxFQUFFLEdBQUUsQ0FBRyxJQUFFLENBQUU7RUFBRTtBQUFBIn0=
```

Comment:
Sure, it's brittle as hell. Big advantage when test coverage is low is that you don't miss details. Updating the generated code could be as easy as setting an environment variable before running the tests again.

Had pretty good experience with "test everything, even the incidental things" as the initial strategy. You can immediately refactor quickly, and it's hard to right targeted tests without refactoring or abusing mocks and spies (which is often more expensive than no tests..)